scripts/setup-openshift.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+set -e
+
+curl -OL https://github.com/openshift/origin/releases/download/v3.10.0/openshift-origin-client-tools-v3.10.0-dd10d17-linux-64bit.tar.gz
```

Comment:
I see `v3.10.0` in here several times.  Worth putting that in var to make upgrades less error prone?